task-definition.web.json

```diff
@@ -101,6 +101,14 @@
                 {
                     "name": "CLICKHOUSE_DENORMALIZED_PROPERTIES",
                     "value": "issampledevent,currentscreen,objectname"
+                },
+                {
+                    "name": "PLUGIN_SERVER_INGESTION",
+                    "value": "false"
+                },
```

Comment:
This disables plugin server ingestion though, and this PR is literally "Enable PLUGIN_SERVER_INGSTION" 🤔

Reply:
Yeah... :D. To be fair, I'd like to just enable this without the ingestion to make sure the plugin server is able to talk with clickhouse (and we don't get an event lag), before turning it on.